Guldier/settings.py

```diff
@@ -20,22 +20,21 @@
 # Build paths inside the project like this: BASE_DIR / 'subdir'.
 BASE_DIR = Path(__file__).resolve(strict=True).parent.parent
 
-
 # Quick-start development settings - unsuitable for production
 # See https://docs.djangoproject.com/en/3.1/howto/deployment/checklist/
 
 # SECURITY WARNING: keep the secret key used in production secret!
 SECRET_KEY = config['SECRET_KEY']
 
 # SECURITY WARNING: don't run with debug turned on in production!
-DEBUG = False
-
-ALLOWED_HOSTS = ['192.168.17.128', '127.0.0.1']
+DEBUG = True
 
+ALLOWED_HOSTS = ['192.168.17.128', '127.0.0.1', 'localhost']
```

Comment:
Pozbyć się adresów. Django envirion - .env - https://django-environ.readthedocs.io/en/latest/

Reply:
Paweł

Reply:
Done, ale musimy to przegadać.